python/libraries/request.py

```diff
@@ -15,65 +15,114 @@
 
 
 async def get(url, headers):
-    async with aiohttp.ClientSession() as session:
-        async with session.get(url, headers=headers) as r:
-            returndata = string(await r.text())
-            try:
-                returndata = python_to_json(json.loads(await r.text()))
-            except:
-                pass
-            return {"code": r.status, "response": r.reason, "return": returndata}
+    try:
+        async with aiohttp.ClientSession() as session:
+            async with session.get(url, headers=headers) as r:
+                returndata = string(await r.text())
+                try:
+                    returndata = python_to_json(json.loads(await r.text()))
+                except:
+                    pass
+                return {"code": r.status, "response": r.reason, "return": returndata}
+    except aiohttp.client_exceptions.ClientError as err:
+        return {
+            "code": 400,
+            "response": "Cannot connect",
+            "return": string(str(err)),
+        }
```

Comment:
Oh, is this what it returns when it fails? I should make a test for this then

Reply:
What do you mean when it fails, that is the Error, along with the no internet error too

Reply:
Sorry, I didn't realise that you updated the test lol